site/docs/guide/filter-queries.md

```diff
@@ -331,6 +332,7 @@ Example:
 | `"message"`                  | `"message"` | `undefined`  | `undefined` |
 | `"message:entities"`         | `"message"` | `"entities"` | `undefined` |
 | `"message:entities:mention"` | `"message"` | `"entities"` | `"mention"` |
+| `"message:caption_entities:url"` | `"message"` | `"caption_entities"` | `"url"` |
```

Comment:
This is inconsistent with the other rows. The point here is to show how a single query is built up in three parts. Adding a second random query is confusing.

Let's revert.
```suggestion
```